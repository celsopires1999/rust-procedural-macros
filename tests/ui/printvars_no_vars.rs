use printvars::trace_vars;

#[trace_vars()]
fn add(mut a: u32) -> u32 {
    a = a + 1;
    a += 1;
    a
}

fn main() {}
