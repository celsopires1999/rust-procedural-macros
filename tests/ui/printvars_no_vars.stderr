error: printvars requires arguments.
 --> tests/ui/printvars_no_vars.rs:3:1
  |
3 | #[trace_vars()]
  | ^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `trace_vars` (in Nightly builds, run with -Z macro-backtrace for more info)
